src/components/Comments/WriteComment.js

```diff
@@ -152,13 +151,13 @@ export const styles = StyleSheet.create({
   columnContainer: {
     display: 'flex',
     flexDirection: 'column',
-    backgroundColor: colors.white,
+    backgroundColor: COLORS.white,
     flex: 1,
     width: '100%',
     position: 'absolute',
     bottom: 0,
     borderTopWidth: 1,
-    borderTopColor: colors.gray1,
+    borderTopColor: COLORS.gray9,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color and border top color of the `columnContainer` style have been updated. Ensure that these new colors are consistent with your design system.
```diff
- backgroundColor: colors.white,
+ backgroundColor: COLORS.white,
...
- borderTopColor: colors.gray1,
+ borderTopColor: COLORS.gray9,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->